lib/recurly.js

```diff
@@ -486,5 +489,11 @@ function normalizeOptions (options) {
     options.fields.all = { style: baseStyleConfig.all };
   }
 
+  // 4
+  const preflightDeviceDataCollectorOptions =  options?.risk?.threeDSecure?.preflightDeviceDataCollector;
```

Comment:
nit: `options` should be reliably present: `options.risk?.threeDSecure?.preflightDeviceDataCollector;`